setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-drag-and-drop-v2',
-    version='2.5.0',
+    version='2.5.1',
```

Comment:
We're adding a feature, so please bump the minor version. It seems that the `2.6.0` tag already exists, so let's use `2.7.0`.

If you wouldn't mind, please also update the `Changelog.md` with a missing `2.6.0` entry, e.g.
```md
Version 2.6.0 (2022-10-24)
---------------------------

* Add package publishing workflow.
```